batch7crud-roll6/src/main/java/com/lftechnology/batch7crud/db/DbConnector.java

```diff
@@ -0,0 +1,29 @@
+package com.lftechnology.batch7crud.db;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.sql.DriverManager'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)